code/modules/mob/dead/observer/observer.dm

```diff
@@ -905,6 +905,12 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	if(!invisibility)
 		. += "It seems extremely obvious."
 
+/mob/dead/observer/examine_more(mob/user)
+	if(!IsAdminGhost(user, TRUE))
+		return ..()
+	. = list("<span class='notice'><i>You examine [src] closer, and note the following...</i></span>")
```

Comment:
this sounds weird about an ooc action like listing admin commands

Reply:
Yeah but it puts it in context and shows that it comes from the double examine, like other examine_more messages. It looked weirder without it